pkg/components/enterprise.go

```diff
@@ -58,8 +58,8 @@ var (
 	}
 
 	ComponentElasticTseeInstaller = component{
-		Version: "v2.7.0-0.dev-38-gbf021d6",
-		Digest:  "sha256:db5d3ee0a5c659337efaa1e48bd8716b913ace1e745a73de59cfad5d79cfaba3",
+		Version: "v2.8.0",
+		Digest:  "sha256:2509c3a597fba10aea2f847c5e48d224f1a66413c897f07208bef99968f0541d",
```

Comment:
I ran `make gen-versions` and got `sha256:d976ad2505b11841afc35dfaa5ffbc6305addd232a7fd707cd6ab0519d942fc2`
I'm not sure which is correct yet.

Reply:
Yours is correct. 

The problem, is I made a quick script to pull the SHAs out of the images and list all of them in one go. Looks like the listing has some GCR SHAs instead of Quay SHAs. 

I'm going through them again now. 

Thanks for catching this.